server/src/controllers/Pki.js

```diff
@@ -78,6 +78,17 @@ exports.getCurrentCARootCert = function getCurrentCARootCert (req, res, next) {
     });
 };
 
+exports.getEnvStatus = function getEnvStatus (req, res, next) {
+  var envId = req.swagger.params['envId'].value;
```

Comment:
it needs to receive also the participant

Reply:
Changed per review. Thank you